content/terraform-plugin-testing/v1.13.x/docs/plugin/testing/testing-patterns.mdx

```diff
@@ -200,6 +201,61 @@ tests is sufficient as opposed to having two separate tests.
 
 [Back to top](#table-of-contents)
 
+## Import mode: verify that import and create produce the same state
+
+To exercise a provider's import behaviors, the test author adds a test step in import mode.
+In this example, `ImportState: true` indicates an import mode test step.
+
+```go
+func TestAccExampleWidget_update(t *testing.T) {
```

Comment:
💭 This is taking up a lot of vertical space. Maybe omit some of the step 1 / step 2 details. Trade-off: a reader cannot copy-paste-run the test. Although that is likely already true.

A link to a fully working "textbook" example would be super nice.

Reply:
+1 for a full example over an abbreviated one. Especially since step 3 relies on some of the other steps details.

Reply:
Yeah, all the widget examples here are pretty old, might be worth a separate effort to update all the docs 😆